components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionManagementServiceImpl.java

```diff
@@ -46,8 +47,18 @@ public class ActionManagementServiceImpl implements ActionManagementService {
     private static final ActionManagementService INSTANCE = new ActionManagementServiceImpl();
     private static final CacheBackedActionMgtDAO CACHE_BACKED_DAO =
             new CacheBackedActionMgtDAO(new ActionManagementDAOImpl());
+    private static final ActionValidator ACTION_VALIDATOR = new ActionValidator();
     private static final ActionSecretProcessor ACTION_SECRET_PROCESSOR = new ActionSecretProcessor();
 
+    private static final String ACTION_NAME_FIELD = "Action name";
+    private static final String ENDPOINT_AUTHENTICATION_URI_FIELD = "Endpoint authentication URI";
+    private static final String ENDPOINT_AUTHENTICATION_TYPE_FIELD = "Endpoint authentication type";
+    private static final String USERNAME_FIELD = "Username";
+    private static final String PASSWORD_FIELD = "Password";
+    private static final String ACCESS_TOKEN_FIELD = "Access token";
+    private static final String API_KEY_HEADER_FIELD = "API key header name";
+    private static final String API_KEY_VALUE_FIELD = "API key value";
```

Comment:
Shall we define these constants in the ActionMgtConstants class. `ACTION_NAME_FIELD` and `ENDPOINT_AUTHENTICATION_URI_FIELD` seems to be duplicated in ActionValidator class as well

Reply:
Updated with 487a8d524e3dd6f8618595b7b9d9266bd19f8c59